lib/fizzy/parser_expr.cpp

```diff
@@ -619,7 +619,12 @@ parser_result<Code> parse_expr(
         case Instr::i64_store32:
         {
             // alignment
-            std::tie(std::ignore, pos) = leb128u_decode<uint32_t>(pos, end);
+            uint32_t align;
```

Comment:
```suggestion
            uint32_t alignment;
```
and drop `// alignment` comment?

Reply:
It is called `align` in the spec.

Reply:
Yes, drop the comment.

Reply:
Fixed.